uaainvite/templates/email/body.html

```diff
@@ -34,6 +34,11 @@
         <p>
         <ul>
             <li>
+                To complete the registration process, you will need to download a 2-step
+                verification app that supports time-based one-time password (TOTP) - such
+                as Google Authenticator <a href="https://itunes.apple.com/us/app/google-authenticator/id388497605">IOS</a> <a href="https://play.google.com/store/apps/details?id=com.google.android.apps.authenticator2&hl=en">Android.</a>
```

Comment:
Let's be more consistent with the docs: https://github.com/18F/cg-site/pull/539/files#diff-ded3174fabce4781a4473fa773fd45d4R30